packages/opencensus-resource-util/src/resource-utils.ts

```diff
@@ -158,7 +158,8 @@ async function getProjectId() {
 /** Gets instance id from GCP instance metadata. */
 async function getInstanceId() {
   try {
-    return await gcpMetadata.instance('id');
+    const id = await gcpMetadata.instance('id');
```

Comment:
This is numerical ID, need to convert to string before assigning to label value.